src/transformers/convert_marian_to_pytorch.py

```diff
@@ -374,20 +376,21 @@ def __init__(self, source_dir):
         self.state_dict = np.load(npz_path)
         cfg = load_config_from_state_dict(self.state_dict)
         assert cfg["dim-vocabs"][0] == cfg["dim-vocabs"][1]
-        assert "Wpos" not in self.state_dict
+        assert "Wpos" not in self.state_dict, "Wpos key in state dictionary"
         self.state_dict = dict(self.state_dict)
         self.wemb, self.final_bias = add_emb_entries(self.state_dict["Wemb"], self.state_dict[BIAS_KEY], 1)
         self.pad_token_id = self.wemb.shape[0] - 1
         cfg["vocab_size"] = self.pad_token_id + 1
         # self.state_dict['Wemb'].sha
         self.state_keys = list(self.state_dict.keys())
-        if "Wtype" in self.state_dict:
-            raise ValueError("found Wtype key")
+        assert "Wtype" not in self.state_dict, "Wtype key in state dictionary"
```

Comment:
```suggestion
        assert "Wtype" not in self.state_dict, "Wtype key not in state dictionary"
```

Reply:
Are you sure ? it seems to me that the assert clause activates when the key is in the state dict, not the reverse, so that's what the error message should display

Reply:
no you're correct. I don't see the value of the change but my suggestion is horrible.